### ISSUE TITLE
gitGraph doesn't support more than 8 branches

### ISSUE THREAD

**Issue by AssassinLV** [tomoyukim/vscode-mermaid-editor#68]
**Describe the bug**
gitGraph doesn't generate lines for branches when more than 8 branches

**To Reproduce**
```
gitGraph
    commit
    branch aaa
        commit
    branch sss
        commit
    branch ddd
        commit
    branch fff
        commit
    branch ggg
        commit
    branch hhh
        commit
    branch jjj
        commit
    branch kkk
        commit
    branch lll
        commit
```

**Expected behavior**
![image](https://user-images.githubusercontent.com/30855600/199421647-f8cb0612-f5c8-4251-b40b-d14a8cc6d826.png)

**Screenshots / Actual behavior**
![image](https://user-images.githubusercontent.com/30855600/199421435-f329626d-ca63-456e-b0a0-1f2d7d1f6c00.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - VSCode version: 1.72.2
 - Extension version: 0.15.0

**Comment by tomoyukim** [comment]
Thank you for your bug report. I found it was fixed at mermaid.js v 9.2.0. I'll update the dependencies soon.

**Comment by tomoyukim** [comment]
I released v0.16.1 with the latest mermaid.js. Thank you for waiting.
